tests/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I can try to config ts-lint work with prettier dictionary, not sure it can consume rules defined in `.prettierrc`

There are two additional steps required to make prettier work with VSCode properly: 
1. Set `Format On Save` in settings to true;
2. Restart the VSCode and reopen the project

The location of the file is correct; I assume the VSCode will apply the settings from the file to the file tree starting from the folder with the `.prettierrc`. The settings from the file will override settings in the IDE.

Reply:
ok let me try that. and will confirm

Reply:
confirmed, enabling "Format on Save" and making changes to the `.prettierrc` file in its current location is taking affect.